src/Generator/Generators/CSharp/CSharpTextTemplate.cs

```diff
@@ -2573,11 +2598,11 @@ public struct ParamMarshal
         {
             if (param.Type is BuiltinType)
             {
-                return new ParamMarshal { Name = param.Name, Param = param };
+                return new ParamMarshal { Name = param.Name, Param = param, HasFixedBlock = false };
```

Comment:
@tritao said he wanted to review your code too a little later. In the mean time - since you have to wait anyway - you might skip setting HasFixedBlock to 'false' because Boolean fields are false by default.